proto/buf/alpha/audit/v1alpha1/envelope.proto

```diff
@@ -531,6 +531,7 @@ message Event {
   // of the event.
   oneof actor {
     UserActor user = 2;
+    ServiceActor service_actor = 59;
```

Comment:
@mfridman i updated to ServiceActor as suggested, however, i'm not going to change field 5 to `service_name` and the new field as `service` because even though a field rename is not typically a breaking change it would be in this case because these entries are stored as JSONB fields in postgres (table structure example below). the inconsistency in the actor field naming grosses me out a little bit, what do you think?
```
CREATE TABLE audit_logs (
  ...
  event JSONB NOT NULL,
  ...
);
```